core/src/main/java/org/apache/accumulo/core/file/rfile/bcfile/Compression.java

```diff
@@ -231,14 +246,26 @@ public OutputStream createCompressionStream(OutputStream downStream, Compressor
        */
       private static final int DEFAULT_BUFFER_SIZE = 32 * 1024;
 
+      private final Boolean USE_QAT =
+          Boolean.valueOf(System.getProperty(USE_QAT_PROPERTY, "false"));
```

Comment:
Since this will return something non-null, you can assign it to a primitive `boolean`. Also, to support `-Duse.qat` in addition to `-Duse.qat=true`, this logic will need to be adjusted a bit. Could see how Maven does it, because it supports similar flags like `-DskipTests` where you don't have to specify `-DskipTests=true`.